bin/GameIndex.yaml

```diff
@@ -33827,7 +33827,7 @@ SLUS-20702:
   region: "NTSC-U"
   compat: 5
 SLUS-20703:
-  name: "Air Force - Delta Strike"
+  name: "AirForce: Delta Strike"
```

Comment:
bit difficult because the cover is in all caps, but generally Airforce is one word, no capitalisation in the middle

Example: https://www.covercentury.com/covers/ps2/e/energy_airforce_-_aim_strike_d.jpg